.circleci/config.yml

```diff
@@ -0,0 +1,68 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/configuration-reference
+version: 2.1
+orbs:
+  gh: circleci/github-cli@2.2.0
+# Define a job to be invoked later in a workflow.
+# See: https://circleci.com/docs/configuration-reference/#jobs
+jobs:
+  update-nvrf:
+    # Specify the execution environment. You can specify an image from Docker Hub or use one of our convenience images from CircleCI's Developer Hub.
+    # See: https://circleci.com/docs/configuration-reference/#executor-job
+    docker:
+      - image: cimg/base:stable
+    # Add steps to the job
+    # See: https://circleci.com/docs/configuration-reference/#steps
+    environment:
+      URL: https://www.eac.gov/sites/default/files/eac_assets/1/6/Federal_Voter_Registration_ENG.pdf
+      NVRF_FILE_NAME: Federal_Voter_Registration_ENG.pdf
+      NEW_BRANCH_NAME: upload-nvrf-pdf
+    steps:
+      #checkout repo and install gh
+      - checkout
+      - gh/install
+      #configure git
+      - run:
+          name: Configure git
+          command: |
+            git config user.email "circleci@df52e93b2563.(none)"
+            git config user.name "Pipeline Commit"
+      # git push origin --delete $NEW_BRANCH_NAME
+      # git checkout -b $NEW_BRANCH_NAME
+      #download pdf from
```

Comment:
I see what you are doing with these but i think you can remove these type of comments as well. The good thing about circle is the steps help break it down but ok to leave them as well if you feel they are needed.

Reply:
this is for lines 30-31